packages/react-components/src/components/ChatMessage/ChatMessageActionsFlyout.tsx

```diff
@@ -147,7 +147,16 @@ export const ChatMessageActionFlyout = (props: ChatMessageActionFlyoutProps): JS
         subMenuProps: {
           id: 'chat-composite-message-contextual-menu-read-name-list',
           items: messageReadByList ?? [],
-          calloutProps: preventUnwantedDismissProps
+          calloutProps: preventUnwantedDismissProps,
+          styles: concatStyleSets({
```

Comment:
I notice some places in the code they define the styles separately in their own variable?

Reply:
In this same file they are also using concatStyle so for consistency I did the same here. And I think it make sense for certain styles that are used multiple places to be a variable.